app/views/govuk_publishing_components/components/_input.html.erb

```diff
@@ -25,13 +25,15 @@
     aria_described_by = aria_described_by.join(" ")
   end
 
+
+puts label
```

Comment:
is this intentional?

Reply:
No, it was forgotten there. @tijmenb could you remove this please?